app/src/main/res/layout/dialog_challenge.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/challenger_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:layout_margin="8dp" />
+
+    <Spinner
+        android:id="@+id/challenger_format"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:layout_margin="8dp" />
+
+    <Spinner
+        android:id="@+id/challenger_team_spinner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:layout_margin="8dp" />
+
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Just use AlertDialog for positive and negative button. You can use `setView` and `View.inflate()` to change AlertDialog view using the above TextView/Spinner